### ISSUE TITLE
Potential error in nuscenes_dataset

### ISSUE THREAD

**Issue by willyd** [traveller59/second.pytorch#226]
Hi, 

@traveller59  Thanks for making this code available.

Could you please confirm that the behavior of this [line](https://github.com/traveller59/second.pytorch/blob/master/second/data/nuscenes_dataset.py#L185) is intended?

It looks like your are using the timestamp in place of the intensity. Is that correct? I think it should be
```python
points = np.concatenate(sweep_points_list, axis=0)[:, [0, 1, 2, 3]]
```

**Comment by poodarchu** [comment]
time stamp is used to estimate velocity, so it's right; 
and of course you can use both intensity and timestamp by indexing [0, 1, 2, 3, 4].

**Comment by cvnerds** [comment]
In the nuScenes paper not only d they write about the dataset, but also an idea to use points from multiple sweeps in order to . For instance tracking a moving point cloud solves the orientation and front/back ambiguities. It appears this could potentially be learned end-to-end simply by providing those timestamps.

**Comment by yinjunbo** [comment]
@willyd  I agree with you. The usage of the timestamp and the intensity should be not contradictory. The author use `points = np.concatenate(sweep_points_list, axis=0)[:, [0, 1, 2, 4]]`, but not `points = np.concatenate(sweep_points_list, axis=0)[:, [0, 1, 2, 3]]`, which is confusing.

**Comment by pyun-ram** [comment]
@cvnerds Thanks for your answer. The use of timestamps as features can be considered as training moving point clouds.
But I still have a concern such an operation will also cause the overfitting of this dataset in an unreasonable manner.
as https://github.com/traveller59/second.pytorch/issues/254#issuecomment-671252738
